### ISSUE TITLE
stop training

### ISSUE THREAD

**Issue by mzhda** [deepinsight/insightface#1475]
I m training arcface, my current situation is as follow:

![image](https://user-images.githubusercontent.com/19206896/115679693-83971480-a35b-11eb-8171-c5c65c392040.png)

![image](https://user-images.githubusercontent.com/19206896/115679913-c0630b80-a35b-11eb-969d-59f705553bf7.png)
 
my learning rate is (0.000001)

also my (acc, lossvalue) looks like stable and does not change.

is it correct for training or I have to stay for more computation

**Comment by Jaypraveen** [comment]
**Unable to find MS1MV2-Arcface training set** 
Please kindly share the "MS1MV2-Arcface" training set link.
Thank you
